lib/provider.js

```diff
@@ -79,6 +79,51 @@ class StorageProvider {
   select(query) {
     throw new GSError(errorCodes.NOT_IMPLEMENTED);
   }
+
+  // Check that given user has acces  to execute a function
+  //   action - string, name of an Action to execute
```

Comment:
I think you can use the new documentation comment format to reduce the number of changes when refactoring all of the comments in the project.